pages/about.jsx

```diff
@@ -32,23 +32,15 @@ var OurFocus = React.createClass({
           <Illustration width={199} height={199}
           src1x="/img/pages/about/about-illustration.svg"
           alt="A graphic showing an illustration of a person, a book, some gears, a map pin, and a square acedemic cap">
-            <h2>Our Focus</h2>
             <ul>
               <li>
-                We focus on peer learning that is production-centered, rooted in open practices, facilitated online and
-                in person, and localized culturally but connected globally.
+                Learn and deepen your 21st Century digital skills, and get better at teaching them to others.
               </li>
               <li>
-                We offer training and event models to help you develop and teach web literacy skills and connect with
-                peers.
+                Contribute to real-world products and projects that are open, participatory and networked.
               </li>
               <li>
-                Our curriculum and web literacy programs are modular, adaptable, and have been developed and tested in
-                various learning settings.
-              </li>
-              <li>
-                We support Hive Learning Networks, city-based communities where educators, organizations and learners
-                network for greater impact.
+                Access free resources like event planning guides and step-by-step teaching activities, or share your own resources to receive feedback from peers Develop your skills in facilitative leadership, participatory practices and collaboration.
```

Comment:
looks like a missing item break: "from peers Develop", that "Develop" looks like a fourth bullet point, not a continued sentence